### PR TITLE
Properly handle the file list for updating the dashboard.

### PR DESCRIPTION
**Pull request by Ferroin**
##### Summary

SSIA

##### Component Name

area/ci

##### Test Plan

n/a

### CODE REVIEW

File: web/gui/bundle_dashboard.py
Comment (vkalintiris) [inline web/gui/bundle_dashboard.py:115]: I take it that the list of files is and will remain small enough so that we can stay below the system's environment size limit.

**Review by vkalintiris — APPROVED** [review]

File: web/gui/bundle_dashboard.py
Comment (Ferroin) [reply web/gui/bundle_dashboard.py:115]: For the moment, it seems exceedingly unlikely that we will ever have a list of file names that takes up more than 128 kiB given that the list is currently at approximately 5.2 kiB.

**Comment by Ferroin** [comment]
@kaskavel @jacekkolasa it would be appreciated if I could get a second review on this so we can merge it and have working automatic PRs for dashboard releases again.

**Review by kaskavel — APPROVED** [review]

**Pull request merged**
